MeshLib/MeshEditing/RasterDataToMesh.cpp

```diff
@@ -0,0 +1,98 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2020, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#include "RasterDataToMesh.h"
+
+#include "MeshLib/Node.h"
+#include "MeshLib/Elements/Element.h"
+
+namespace MeshLib
+{
+
+namespace RasterDataToMesh
+{
+
+bool checkMesh2D(MeshLib::Mesh const& mesh)
```

Comment:
I'd prefer all functions not exposed in the interface to have internal linkage. Either `static` or anonymous namespace.
Makes life of a linker easier.